projects/wave-core/src/lib/layers/symbology/symbology-editor/symbology-editor.component.ts

```diff
@@ -23,20 +25,26 @@ export class SymbologyEditorComponent implements OnDestroy {
     @Input() layer: Layer<AbstractSymbology> = undefined;
     validLayers: Array<Layer<AbstractSymbology>> = undefined;
     private subscriptions: Array<Subscription> = [];
+    private layerChanges = new ReplaySubject<[Layer<AbstractSymbology>, AbstractSymbology]>(1);
 
     constructor(
+        private config: Config,
         public projectService: ProjectService
     ) {
-        const sub = this.projectService.getLayerStream().subscribe(projectLayers => this.validLayers = projectLayers);
-        this.subscriptions.push(sub);
+        const sub1 = this.projectService.getLayerStream().subscribe(projectLayers => this.validLayers = projectLayers);
+        this.subscriptions.push(sub1);
+        const sub2 = this.layerChanges.pipe(debounceTime(config.DELAYS.DEBOUNCE)).subscribe(
```

Comment:
I would suggest either giving the subscriptions a meaningful name or directly wrapping the expression in the `push`.